Makefile

```diff
@@ -7,11 +7,13 @@ LOG_LEVEL = INFO
 PYTHONIOENCODING=utf8
 TESTDIR = tests
 
-SPHINX_APIDOC = 
+SPHINX_APIDOC =
 
 BUILD_ORDER = ocrd_utils ocrd_models ocrd_modelfactory ocrd_validators ocrd_network ocrd
+reverse = $(if $(wordlist 2,2,$(1)),$(call reverse,$(wordlist 2,$(words $(1)),$(1))) $(firstword $(1)),$(1))
```

Comment:
I'm reminded of "intro to functional programming" 20 years ago 👍

Reply:
just plain old recursion – not even a difflist!

I later realised this is not even necessary, since `pip uninstall` seems to be "all at once" (in contrast to `pip install`).